apps/src/JsDebuggerUi.js

```diff
@@ -337,6 +356,34 @@ var boundMouseMoveHandler;
 /** @type {string} */
 var mouseMoveTouchEventName;
 
+/** @type {boolean} */
+let draggingWatchersResizeBar = false;
+
+/** @type {function} */
+let boundMouseMoveHandlerWatchers;
+
+let watchersReferences = {};
+
+function getWatchersElements() {
+  if (!watchersReferences.watchersResizeBar) {
+    watchersReferences.watchersResizeBar = document.getElementById('watchersResizeBar');
+  }
+  if (!watchersReferences.watchersDiv) {
+    watchersReferences.watchersDiv = document.getElementById('debug-watch');
+  }
+  if (!watchersReferences.watchersHeaderDiv) {
+    watchersReferences.watchersHeaderDiv = document.getElementById('debug-watch-header');
+  }
+  if (!watchersReferences.debugConsoleDiv) {
+    watchersReferences.debugConsoleDiv = document.getElementById('debug-console');
+  }
+  return watchersReferences;
```

Comment:
Believe the shorthand is still ruby-only, was considering doing e.g.:

`watchersReferences.debugConsoleDiv = watchersReferences.debugConsoleDiv || document.getElementById('debug-console');`

... maybe that makes it more obvious what's going on. Think I'll do it that way!